tests/utils/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as render } from '@/tests/utils/render'
+export { default as renderWithQueryClient } from '@/tests/utils/renderWithQueryClient'
```

Comment:
It does not seem necessary to manage in the index file, but what do you think? 🤔

e.g.
```tsx
// src/hooks/useAuth.test.tsx
import { renderWithQueryClient } from '@/tests/utils/renderWithQueryClient'
```

Reply:
@jinmayamashita At the beginning time of this PR, I used your recommending syntax `import { renderWithQueryClient } from '@/tests/utils/renderWithQueryClient'`
But after writing test for some files, I had to import the render utility again and again.
I think it's better to simplify the import statement by exporting them in the index file.
So developers does not need to type extra renderWithQueryClient or render after `@/tests/utils`
It's good for lazy developers like me 😅

Reply:
> So developers does not need to type extra renderWithQueryClient or render after @/tests/utils
It's good for lazy developers like me 😅

@mvn-luanvu-hn 

I am a different kind of lazy man as the number of files under the util folder grows, 
I have a hard time getting the util/index file to update along with it. 😅

But I understand your thoughts 👍🏼

Let's try to manage it in the index file and if another issue comes up, let's discuss it again!